10.10.0/Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN npm install -g pm2 \
      && apk add bash \
      && cd /opt/startup \
      && npm install \
+     && npm install npm@latest -g \
```

Comment:
Are you sure you really want the latest always? This might add more risk to customers looking to launch their apps -- their apps might fail to come up due to NPM issues in the 'latest' version..

Reply:
I recall adding this for a workaround that is no longer needed
we should remove it

Reply:
It could always look for `engines.npm` in package.json for which version to use then fall back to `@latest`. Seems safer for situations like what Dhanvi mentioned.

Reply:
Yes. We should avoid adding drfit to module versions in our node images between customer launches as that would mean that we are adding risk to their deployments within Azure when things work in some cases but not in some cases.  

This assumes that we *always* launch exact same image for an app after the initial app creation (e.g on scale-up, etc.).  Is that the case **OR** do we pull in the latest version of the Node image? If we are pulling in the latest version of Node images, that we are already exposing our customers to version drifts between images.